src/options/options_handler.cpp

```diff
@@ -400,5 +401,13 @@ void OptionsHandler::showTraceTags(const std::string& flag, bool value)
   printTags(Configuration::getTraceTags());
 }
 
+void OptionsHandler::strictParsing(const std::string& flag, bool value)
+{
+  if (value)
+  {
+    d_options->write_parser().parsingMode = options::ParsingMode::STRICT;
+  }
```

Comment:
As discussed, I believe we need to set to DEFAULT if the value is false *and* the parsingMode is currently STRICT.

Reply:
Hold on, something went wrong here, it seems that I merged two branches, this shouldn't be in this PR. Let me fix this.